templates/ocs-deployment/cluster_new.yaml

```diff
@@ -0,0 +1,130 @@
+#################################################################################################################
```

Comment:
@vasukulkarni @ebondare shouldn't we move this under CSI dir and call it cluster.py?

Reply:
@ebondare how are you using this one? Are you modifying config and reloading without jinja? If so I think the best place is CSI/ folder

Reply:
@ebondare ?

Reply:
@petr-balogh we will have to use this in deployment only then its going to be useful?

Reply:
> @ebondare how are you using this one? Are you modifying config and reloading without jinja? If so I think the best place is CSI/ folder

As a matter of fact, the testcase I needed this file for was taken from me, so I don't need it anymore. I'll just close this pull request, right?

Reply:
Yep, please close, if we will start using files without templates in deployment we will update our file or update. Thanks